qa/tasks/check_counter.py

```diff
@@ -80,13 +84,29 @@ def end(self):
                 else:
                     log.debug("Getting stats from {0}".format(daemon_id))
 
-                manager = self.ctx.managers[cluster_name]
-                proc = manager.admin_socket(daemon_type, daemon_id, ["perf", "dump"])
-                response_data = proc.stdout.getvalue().strip()
+                if daemon_type == 'mds':
+                    mds_info = status.get_mds(daemon_id)
+                    if not mds_info:
+                        continue
+                    mds = f"mds.{mds_info['name']}"
```

Comment:
Aren't we going forward with querying mds.<gid> (@batrick's proposal)?

Reply:
Yes, the same thing implemented with `tell`. Much better now.

Reply:
Sure, but we need to query the MDS gid rather than using its name.

Reply:
I got your question now. 
I've done this initially: https://github.com/ceph/ceph/commit/9292e38da55007bb5f8eff44d561dbad13513ba4. But then changed it to querying the name because the existing/incoming variable `daemon_id` was not a gid. It's the name retrieved from https://github.com/ceph/ceph/blob/main/qa/cephfs/clusters/1a11s-mds-1c-client-3node.yaml. Does `daemons.items()` provide gid?

Reply:
Check this out: https://github.com/ceph/ceph/blob/main/qa/tasks/check_counter.py#L65. It's written based on the names.

Reply:
> Check this out: https://github.com/ceph/ceph/blob/main/qa/tasks/check_counter.py#L65. It's written based on the names.

Right, its written that way now, but we should be using GID based query as @batrick mentioned.

Reply:
(unresolving conversation)

The MDS tell command should be performed with the MDS gid.

Reply:
@vshankar Fixed